Misc/NEWS.d/next/Library/2023-11-25-20-29-28.gh-issue-112405.cOtzxC.rst

```diff
@@ -0,0 +1 @@
+Optimise :meth:`pathlib.Path.relative_to`. Patch by Alex Waygood.
```

Comment:
```suggestion
Optimize :meth:`pathlib.Path.relative_to`. Patch by Alex Waygood.
```

very nitpicky, and I don't personally mind about British vs American spelling, but "optimize" appears over 300 times in the repo, while "optimise" appears only 7 times :)

Reply:
Booo, both Barney and I are Brits!